### PR TITLE
Added post on 3.0 plugins that GA'd!

### PR DESCRIPTION
**Pull request by bmbouter**
Here is a rendered version: https://bmbouter.github.io/pulpproject.org/2019/12/12/pulp-3-is-GA/

### CODE REVIEW

File: _posts/2019-12-12-pulp-3-is-GA.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+pulp_file can sync, organize, upload, and distribute File content.
+
+
+## Installation
+
+See the installation docs (links below) for each plugin on how to install. They all use the same
+installer so they are very similar. Plugins are capable of being installed all together on one
+installation.
+
+* [pulp_rpm installation]() to manage RPM content
```
Comment (CodeHeeler) [inline _posts/2019-12-12-pulp-3-is-GA.md]: Missing link, probably this one: https://pulp-rpm.readthedocs.io/en/latest/installation.html

**Comment by RCMariko** [comment]
Would a quick explanation of the 3 different version #s for the plugins be helpful?

File: _posts/2019-12-12-pulp-3-is-GA.md (hunk 1 above)
Comment (bmbouter) [reply _posts/2019-12-12-pulp-3-is-GA.md]: Fixed, ty

File: _posts/2019-12-12-pulp-3-is-GA.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+You can upload the following to pulp_rpm:
+
+* RPM content
+* SRPM
+* Advisory
+
+### Container Features
+
+pulp_container can sync, organize, upload and distribute Docker content
```
Comment (RCMariko) [inline _posts/2019-12-12-pulp-3-is-GA.md]: Add period at end of sentence.

File: _posts/2019-12-12-pulp-3-is-GA.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+### File Content
```
Comment (daviddavis) [inline _posts/2019-12-12-pulp-3-is-GA.md]: The other sections are titled "<plugin type> Features". Not sure if you meant to be inconsistent?

**Review by RCMariko — APPROVED** [review]

File: _posts/2019-12-12-pulp-3-is-GA.md (hunk 3 above)
Comment (bmbouter) [reply _posts/2019-12-12-pulp-3-is-GA.md]: I did not intend to be. I updated it. Ty

File: _posts/2019-12-12-pulp-3-is-GA.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+### RPM Features
+
+pulp_rpm can sync, organize, and distribute the following content types currently:
+
+* RPMs
+* SRPMs
+* Advisory Content, i.e. Errata
+* Distribution Trees (kickstart trees)
+* Yum Metadata, e.g. comps.xml
+* moduleMD content
```
Comment (goosemania) [inline _posts/2019-12-12-pulp-3-is-GA.md]: maybe `modularity content` sounds more user friendly. Up to you

File: _posts/2019-12-12-pulp-3-is-GA.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+installation.
+
+* [pulp_rpm installation](https://pulp-rpm.readthedocs.io/en/3.0/installation.html)
+* [pulp_container installation](https://pulp-container.readthedocs.io/en/1.0/installation.html)
+* [pulp_file installation](https://pulp-file.readthedocs.io/en/0.1.0/installation.html)
+
+
+## Quickstart
+
+Pulp_rpm offers a [quickstart guide](https://pulp-rpm.readthedocs.io/en/3.0/installation.html) which
```
Comment (goosemania) [inline _posts/2019-12-12-pulp-3-is-GA.md]: link to a quickstart https://pulp-rpm.readthedocs.io/en/3.0/quickstart.html

File: _posts/2019-12-12-pulp-3-is-GA.md (hunk 4 above)
Comment (bmbouter) [reply _posts/2019-12-12-pulp-3-is-GA.md]: fixed in next push. ty!

File: _posts/2019-12-12-pulp-3-is-GA.md (hunk 2 above)
Comment (bmbouter) [reply _posts/2019-12-12-pulp-3-is-GA.md]: fixed in next push, ty!

File: _posts/2019-12-12-pulp-3-is-GA.md (hunk 5 above)
Comment (bmbouter) [reply _posts/2019-12-12-pulp-3-is-GA.md]: fixed, ty!

**Comment by bmbouter** [comment]
I also added a section explaining the version numbers.

File: _posts/2019-12-12-pulp-3-is-GA.md [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+
+
+## Why these version numbers?
+
+These Pulp plugins use [semantic versioning](https://semver.org/), so the different version numbers
+have different meanings summarized below.
+
+* pulp_rpm is 3.0 to indicate future releases will not contain API breaking changes, and it's the
+  next major release after pulp_rpm 2.y
+* pulp_container uses 1.0 to indicate future releases will not contain API breaking changes, but it
```
Comment (daviddavis) [inline _posts/2019-12-12-pulp-3-is-GA.md:78]: s/it/it is/

Comment (bmbouter) [reply _posts/2019-12-12-pulp-3-is-GA.md:78]: fixed in next PR https://github.com/pulp/pulpproject.org/pull/230

**Pull request merged**